### DISCUSSION TITLE
gross shading

### DISCUSSION THREAD

**Discussion by genbread** [bnpr/Malt#164] [Q&A]
coming back to malt after a long while and dont get why these shadows look gross 
turning off self shadows removes it but i want them on
the object is shaded smooth too
![image](https://user-images.githubusercontent.com/82292471/141515773-5e7e5b0f-1b34-4fe3-884e-8a7451804206.png)

**Comment by pragma37** [comment]
Could you post the blend file so I can look into it?

**Reply by JPLost** [reply]
[Malt_Testing.zip](https://github.com/bnpr/Malt/files/7540966/Malt_Testing.zip)

Here you go, Blender 3.0.0 - Beta file.
If the gradient slider goes over value 0.500, it breaks the shading.

**Reply by pragma37** [reply]
Thanks!
Should be fixed now.
